.travis/run_task.py

```diff
@@ -48,14 +48,15 @@ def main():
         print("*** We don't need the publish task, exiting early")
         return 0
 
-    if task == 'loris-build':
-        publish_task = 'loris-publish'
-    elif task == 'miro_preprocessor-test':
-        publish_task = 'miro_preprocessor-publish'
-    elif task == 'lambdas-test':
-        publish_task = 'lambdas-publish'
-    elif task == 'monitoring_lambdas-test':
-        publish_task = 'monitoring_lambdas-publish'
+    if task in [
```

Comment:
Much nicer. Still makes me feel like i'd like the projects themselves to specify the build steps they require.

Reply:
Yep, although soon deploy will be expunged entirely.